### PR TITLE
[1LP][RFR] smoke tests for api's event_streams collection

### PR DESCRIPTION
**Pull request by izapolsk**
{{pytest: -k 'test_find_created_events' cfme/tests/test_rest.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit efb256bc13de98a3c87b25f97c5f33a59d5852ec:

`cfme/tests/test_rest.py`:
- :red_circle: Line 920:9: **F841** *local variable 'collection' is assigned to but never used*
- :heavy_exclamation_mark: Line 940:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a5675efe5f5478594768b824964607a1c1a46fe8:

`cfme/tests/test_rest.py`:
- :red_circle: Line 915:9: **F841** *local variable 'collection' is assigned to but never used*
- :heavy_exclamation_mark: Line 935:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 93e9f61480b2f0d5b4ba8aae7de7f2e3dfa634d5:

`cfme/tests/test_rest.py`:
- :red_circle: Line 915:9: **F841** *local variable 'collection' is assigned to but never used*
- :red_circle: Line 932:13: **F841** *local variable 'ems_event_type' is assigned to but never used*
- :red_circle: Line 933:13: **F841** *local variable 'ems_events' is assigned to but never used*
- :red_circle: Line 939:13: **F841** *local variable 'miq_event_type' is assigned to but never used*
- :red_circle: Line 940:13: **F841** *local variable 'miq_events' is assigned to but never used*
- :heavy_exclamation_mark: Line 951:1: **W391** *blank line at end of file*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ae57b073d897c77f51138489e9fc192a0e3185e9:

`cfme/tests/test_rest.py`:
- :red_circle: Line 915:9: **F841** *local variable 'collection' is assigned to but never used*
- :red_circle: Line 932:13: **F841** *local variable 'ems_event_type' is assigned to but never used*
- :red_circle: Line 933:13: **F841** *local variable 'ems_events' is assigned to but never used*
- :red_circle: Line 939:13: **F841** *local variable 'miq_event_type' is assigned to but never used*
- :red_circle: Line 940:13: **F841** *local variable 'miq_events' is assigned to but never used*
- :heavy_exclamation_mark: Line 951:1: **W391** *blank line at end of file*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 295afd6c7846578d54cb17697be34eb23946367d:

`cfme/tests/test_rest.py`:
- :bangbang: Line 963:42: **E127** *continuation line over-indented for visual indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/test_rest.py [hunk 1]
```diff
@@ -906,3 +911,63 @@ def test_delete_notifications_from_collection(self, appliance, generate_notifica
...
+            test_flag: rest
+        """
+        vm_name = vm_modscope
+        collections = appliance.rest_api.collections
+        vm_id = collections.vms.get(name=vm_name).id
+
+        ems_event_type = 'EmsEvent'
+        if a_provider.one_of(RHEVMProvider):
+            ems_events = [
+                ('create', {'event_type': 'USER_ADD_VM_FINISHED_SUCCESS'}),
```
[the remark this answers is not in the document]
Comment (izapolsk) [reply cfme/tests/test_rest.py]: I'd rather create vms of each type instead. In either way I moved events to provider classes for some time.

**Comment by dajoRH** [comment]
Lint report for commit c28f51db3ee91851440a1e35fac3d03c9c81fa68:

`cfme/common/provider.py`:
- :heavy_exclamation_mark: Line 60:1: **W293** *blank line contains whitespace*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**